apps/client/src/locales/messages.fr.xlf

```diff
@@ -3027,6 +3027,7 @@
       </trans-unit>
       <trans-unit id="6268646680388419543" datatype="html">
         <source>Emergency Fund</source>
+        <target state="translated">Fonds d'Urgence</target>
```

Comment:
`<target state="translated">Fonds d&apos;Urgence</target>`

For consistency...